cloudsmith/data_source_package.go

```diff
@@ -0,0 +1,185 @@
+package cloudsmith
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path"
+
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func dataSourcePackageRead(d *schema.ResourceData, m interface{}) error {
+	pc := m.(*providerConfig)
+	namespace := requiredString(d, "namespace")
+	repository := requiredString(d, "repository")
+	identifier := requiredString(d, "identifier")
+	download := d.Get("download").(bool)
+	outputPath := d.Get("output_path").(string)
```

Comment:
can you use `requiredBool` and `requiredString` for these? they do the same thing and would keep things consistent